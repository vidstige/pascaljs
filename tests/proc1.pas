program Sample;
procedure Output(A,B: Integer; C: String);
begin
    WriteLn(A, ', ', B, ', ', C); 
end;
begin
  Output(7, 8, 'C');
end.