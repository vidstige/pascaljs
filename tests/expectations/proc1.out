7, 8, C
